www/profile/inner.js

```diff
@@ -374,7 +407,7 @@ define([
 
         var $delButton = $('<button>', {
             'class': 'cp-app-profile-avatar-delete btn btn-danger fa fa-times',
-            title: Messages.fc_delete
+            title: Messages.profile_remove_avatar //XXX
```

Comment:
No need for XXX here if you've already added the temporary key to customize.dist/messages.js